src/layouts/EditHomepage.jsx

```diff
@@ -295,43 +333,10 @@ const EditHomepage = ({ match }) => {
 
   useEffect(() => {
     if (scrollRefs.length > 0) {
-      scrollRefs[frontMatter.sections.length - 1].current.scrollIntoView()
+      scrollTo(scrollRefs[frontMatter.sections.length - 1])
     }
   }, [scrollRefs, frontMatter.sections.length])
```

Comment:
going to [this](https://cms.isomer.gov.sg/sites/a-test-v4/homepage) site has the same behaviour on production.

this was never noticeable because we instantly snapped there but now cos of smooth scrolling, we do know @__@

cc @jandonyada and @sehyunidaaa if this should be fixed (imo, yes if easy but can drop if hard cos is improvement)

Reply:
removing - will release as separate PR

Reply:
@seaerchin isn't this just doing: 

```
const [isFirstLoadComplete, setIsFirstLoadComplete] = useState(false)

useEffect(() => setIsFirstLoadComplete(true), [])

 useEffect(() => {
    if (scrollRefs.length > 0 && isFirstLoadComplete) {
      scrollTo(scrollRefs[frontMatter.sections.length - 1])
    }
  }, [scrollRefs, frontMatter.sections.length])
```

Reply:
Most likey this doesn't work because the sections are generated from the frontmatter some time after the initial page load - this should work instead

```
const [isFirstLoadComplete, setIsFirstLoadComplete] = useState(false)

  useEffect(() => {
    if (scrollRefs.length > 0) {
      if (!isFirstLoadComplete) {
        setIsFirstLoadComplete(true)
      } else {
        scrollTo(scrollRefs[frontMatter.sections.length - 1])
      }
    }
  }, [scrollRefs, frontMatter.sections.length])
```